schemas/com.appsflyer/attribution/jsonschema/1-0-0

```diff
@@ -0,0 +1,566 @@
+          "type": ["string", "null"],
+          "maxLength": 1024
+    },
+    "af_adset": {
+        "description": "Adset",
+        "type": ["string", "null"],
+        "maxLength": 1024
+    },   
+    "af_adset_id": {
+        "description": "Adset ID",
+        "type": ["string", "null"],
+        "maxLength": 1024
+    },     
+    "af_attribution_lookback": {
+        "description": "Attribution Lookback Window",
+        "type": ["string", "null"],
+        "maxLength": 1024
+    },  
+    "af_c_id": {
+        "description": "Campaign ID",
+        "type": ["string", "null"],
+        "maxLength": 1024
+    }, 
+    "af_channel": {
+        "description": "Channel",
+        "type": ["string", "null"],
+        "maxLength": 1024
+    }, 
+    "af_cost_currency": {
+        "description": "Cost Currency",
+        "type": ["string", "null"],
+        "maxLength": 1024
+    }, 
+    "af_cost_model": {
+        "description": "Cost Model",
+        "type": ["string", "null"],
+        "maxLength": 1024
+    }, 
+    "af_cost_value": {
+        "description": "Cost Value",
+        "type": ["string", "null"],
+        "maxLength": 1024
+    }, 
+    "af_keywords": {
+        "description": "Keywords",
+        "type": ["string", "null"],
+        "maxLength": 1024
+    }, 
+    "af_prt": {
+        "description": "Partner",
+        "type": ["string", "null"],
+        "maxLength": 1024
+    }, 
+    "af_reengagement_window": {
+        "description": "Reengagement Window",
+        "type": ["string", "null"],
+        "maxLength": 1024
+    }, 
+    "af_siteid": {
+        "description": "Site ID",
+        "type": ["string", "null"],
+        "maxLength": 1024
+    }, 
+    "af_sub_siteid": {
+        "description": "Sub Site ID",
+        "type": ["string", "null"],
+        "maxLength": 1024
+    }, 
+    "af_sub1": {
+        "description": "Sub Param 1",
+        "type": ["string", "null"],
+        "maxLength": 1024
+    }, 
+    "af_sub2": {
+        "description": "Sub Param 2",
+        "type": ["string", "null"],
+        "maxLength": 1024
+    }, 
+    "af_sub3": {
+        "description": "Sub Param 3",
+        "type": ["string", "null"],
+        "maxLength": 1024
+    }, 
+    "af_sub4": {
+        "description": "Sub Param 4",
+        "type": ["string", "null"],
+        "maxLength": 1024
+    }, 
+    "af_sub5": {
+        "description": "Sub Param 5",
+        "type": ["string", "null"],
+        "maxLength": 1024
+    }, 
+    "amazon_aid": {
+        "description": "Amazon Fire ID",
+        "type": ["string", "null"],
+        "maxLength": 1024
+    }, 
+    "android_id": {
+        "description": "Android ID",
+        "type": ["string", "null"],
+        "maxLength": 1024
+    }, 
+    "api_version": {
+        "description": "API Version e.g. 2.0",
+        "type": ["string", "null"],
+        "maxLength": 10
+    }, 
+    "app_id": {
+        "description": "App ID",
+        "type": ["string", "null"],
```

Comment:
`app_id` is a required field but is shown here as a allowing nulls. I imagine this should be `"type": "string"` assuming `app_id` is always available from AppsFlyers Push API V2.

Reply:
Fixed. Im not sure if they always send app_id so I removed it.